### PR TITLE
permissions.yml doctordoom

### PR DESCRIPTION
**Pull request by adambash**

### CODE REVIEW

File: PermissionsEx/permissions.yml [hunk 1]
```diff
@@ -2302,6 +2302,24 @@ groups:
...
+  doctordoom:
+    inheritance:
+    - mortal
+    options:
+      prefix: '&2D&ao&2ct&ao&2r &aD&2o&ao&2m'
+      suffix: '&b'
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.kill
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: proof of killing via willpower

File: PermissionsEx/permissions.yml [hunk 2]
```diff
@@ -2302,6 +2302,24 @@ groups:
...
+        - ch.alias.hoveron
+        - ch.alias.hoveroff
+        - ch.alias.empower
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: empowering another

File: PermissionsEx/permissions.yml [hunk 3]
```diff
@@ -2302,6 +2302,24 @@ groups:
...
+        - essentials.gamemode
+        - essentials.gamemode.creative
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: creating matter from nothing

File: PermissionsEx/permissions.yml [hunk 4]
```diff
@@ -2302,6 +2302,24 @@ groups:
...
+        - essentials.skull
+        - essentials.skull.*
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: also creating matter from nothing

File: PermissionsEx/permissions.yml (hunk 1 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: https://video.search.yahoo.com/yhs/search;_ylt=AwrTcdBiKH1YSyAALxQPxQt.?p=Dr.Doom+exploding+heads+in+Fantastic+Four+movie&fr=yhs-omr-001&fr2=piv-web&hspart=omr&hsimp=yhs-001&type=86311436#action=view&id=5&vid=4af20d00b907867b08585f2441bc282a

File: PermissionsEx/permissions.yml (hunk 4 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: https://www.quora.com/What-are-Dr-Dooms-powers  
he is a magical being that can make stuff out of nothing

File: PermissionsEx/permissions.yml (hunk 3 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: https://www.quora.com/What-are-Dr-Dooms-powers
he is a magical being that can create stuff from nothing

File: PermissionsEx/permissions.yml (hunk 2 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: ill take that away

File: PermissionsEx/permissions.yml (hunk 1 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: accepted

File: PermissionsEx/permissions.yml [hunk 5]
```diff
@@ -2302,6 +2302,25 @@ groups:
...
+      prefix: '&2D&ao&2ct&ao&2r &aD&2o&ao&2m'
+      suffix: '&b'
+    worlds:
+      survival:
+        permissions:
+        - ch.alias.kill
+        - ch.alias.hoveron
+        - ch.alias.hoveroff
+        - essentials.tp
+        - essentials.tp.*
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: proof of teleportation

File: PermissionsEx/permissions.yml [hunk 6]
```diff
@@ -2302,6 +2302,25 @@ groups:
...
+        - essentials.gamemode
+        - essentials.gamemode.creative
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: proof of creating matter from nothing

File: PermissionsEx/permissions.yml [hunk 7]
```diff
@@ -2302,6 +2302,25 @@ groups:
...
+        - essentials.skull
+        - essentials.skull.*
```
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: EVIDENCE REQUIRED: also proof of creating matter from nothing

File: PermissionsEx/permissions.yml (hunk 6 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: https://www.quora.com/What-are-Dr-Dooms-powers he is a magical being that can create something out of nothing

File: PermissionsEx/permissions.yml (hunk 5 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: https://www.quora.com/What-are-Dr-Dooms-powers

Comment (malonnnn) [inline PermissionsEx/permissions.yml]: Sorry, does not count. Picture or video only accepted.

File: PermissionsEx/permissions.yml (hunk 6 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: Sorry, does not count. Picture or video only accepted.

File: PermissionsEx/permissions.yml (hunk 5 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: https://images.search.yahoo.com/yhs/search;_ylt=A86.J3dtLn1YvE8AdiMPxQt.?p=doctor+doom+teleporting&fr=yhs-omr-001&fr2=piv-web&hspart=omr&hsimp=yhs-001&type=86311436#id=1&iurl=https%3A%2F%2Fi.imgur.com%2FClODpRN.jpg&action=click

File: PermissionsEx/permissions.yml (hunk 6 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: https://images.search.yahoo.com/yhs/search;_ylt=A86.J3dtLn1YvE8AdiMPxQt.?p=doctor+doom+teleporting&fr=yhs-omr-001&fr2=piv-web&hspart=omr&hsimp=yhs-001&type=86311436#id=3&iurl=http%3A%2F%2Flounge.moviecodec.com%2Fimages%2Fattachment%2Fdr-doom-vs-hst-76437.png&action=click

File: PermissionsEx/permissions.yml (hunk 5 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: accepted

File: PermissionsEx/permissions.yml (hunk 6 above)
Comment (malonnnn) [inline PermissionsEx/permissions.yml]: that is not matter, that is energy. matter is something you can hold. he is using an energy shield.

Comment (adambash) [inline PermissionsEx/permissions.yml]: https://images.search.yahoo.com/yhs/search;_ylt=AwrTcXZ_L31Yw_MAQAM2nIlQ;_ylu=X3oDMTBsZ29xY3ZzBHNlYwNzZWFyY2gEc2xrA2J1dHRvbg--;_ylc=X1MDMTM1MTE5NTcwMgRfcgMyBGFjdG4DY2xrBGJjawNic2c0NXI5YjQxaW1nJTI2YiUzRDQlMjZkJTNET2tYb0hVZHBZRUlrQ0NxeWxOYmg5Xy5RMEliekwxUWxnLjBxdVEtLSUyNnMlM0RpaCUyNmklM0RCOG5lYU00WXU5MHNmN3AueFlkYQRjc3JjcHZpZANKWEs1SURJd05pNi5RSVhhVmtESzBBWXFOamd1TVFBQUFBQXRfY01xBGZyA3locy1vbXItMDAxBGZyMgNzYS1ncARncHJpZAM0aW9tU0Z1ZlRiTzdnQ1BRNGlGa3dBBG10ZXN0aWQDbnVsbARuX3N1Z2cDMARvcmlnaW4DaW1hZ2VzLnNlYXJjaC55YWhvby5jb20EcG9zAzAEcHFzdHIDBHBxc3RybAMEcXN0cmwDMjQEcXVlcnkDZG9jdG9yIGRvb20gZW5lcmd5IHN3b3JkBHRfc3RtcAMxNDg0NTk5MTkzBHZ0ZXN0aWQDbnVsbA--?gprid=4iomSFufTbO7gCPQ4iFkwA&pvid=JXK5IDIwNi6.QIXaVkDK0AYqNjguMQAAAAAt_cMq&p=doctor+doom+energy+sword&fr=yhs-omr-001&fr2=sb-top-images.search.yahoo.com&ei=UTF-8&n=60&x=wrt&type=86311436&hsimp=yhs-001&hspart=omr#id=21&iurl=http%3A%2F%2Fmedinnus.com%2Favengersassemble%2Fimages%2Fdr_doom_01.png&action=click

Comment (adambash) [inline PermissionsEx/permissions.yml]: http://medinnus.com/avengersassemble/images/dr_doom_01.png

File: PermissionsEx/permissions.yml (hunk 7 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: http://medinnus.com/avengersassemble/images/dr_doom_01.png

Comment (adambash) [inline PermissionsEx/permissions.yml]: http://files.animatedsuperheroes.com/images/superherosquad/superherosquad_ep02a.jpg

File: PermissionsEx/permissions.yml (hunk 6 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: http://files.animatedsuperheroes.com/images/superherosquad/superherosquad_ep02a.jpg

Comment (adambash) [inline PermissionsEx/permissions.yml]: https://images.search.yahoo.com/yhs/search;_ylt=AwrTcYRlNn1Yh8QABhE2nIlQ;_ylu=X3oDMTBsZ29xY3ZzBHNlYwNzZWFyY2gEc2xrA2J1dHRvbg--;_ylc=X1MDMTM1MTE5NTcwMgRfcgMyBGFjdG4DY2xrBGJjawNic2c0NXI5YjQxaW1nJTI2YiUzRDQlMjZkJTNET2tYb0hVZHBZRUlrQ0NxeWxOYmg5Xy5RMEliekwxUWxnLjBxdVEtLSUyNnMlM0RpaCUyNmklM0RCOG5lYU00WXU5MHNmN3AueFlkYQRjc3JjcHZpZANmR29BYVRJd05pNi5RSVhhVmtESzBBQm1Oamd1TVFBQUFBQ1hSWklyBGZyA3locy1vbXItMDAxBGZyMgNzYS1ncARncHJpZANPVmx0OE1YTFNuR0trNkZQcTViX0hBBG10ZXN0aWQDbnVsbARuX3N1Z2cDMARvcmlnaW4DaW1hZ2VzLnNlYXJjaC55YWhvby5jb20EcG9zAzAEcHFzdHIDBHBxc3RybAMEcXN0cmwDMzEEcXVlcnkDRG9jdG9yIERvb20gZHVwbGljYXRpbmcgaGltc2VsZgR0X3N0bXADMTQ4NDYwMTAxNAR2dGVzdGlkA251bGw-?gprid=OVlt8MXLSnGKk6FPq5b_HA&pvid=fGoAaTIwNi6.QIXaVkDK0ABmNjguMQAAAACXRZIr&p=Doctor+Doom+duplicating+himself&fr=yhs-omr-001&fr2=sb-top-images.search.yahoo.com&ei=UTF-8&n=60&x=wrt&type=86311436&hsimp=yhs-001&hspart=omr#id=8&iurl=http%3A%2F%2Fblog.mysanantonio.com%2Fgeekspeak%2Ffiles%2F2012%2F03%2Fdoctor_vs__doctor_by_docshaner-d36i641.jpg&action=click

he can create more of himself out of nothing

File: PermissionsEx/permissions.yml (hunk 7 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: https://images.search.yahoo.com/yhs/search;_ylt=AwrTcYRlNn1Yh8QABhE2nIlQ;_ylu=X3oDMTBsZ29xY3ZzBHNlYwNzZWFyY2gEc2xrA2J1dHRvbg--;_ylc=X1MDMTM1MTE5NTcwMgRfcgMyBGFjdG4DY2xrBGJjawNic2c0NXI5YjQxaW1nJTI2YiUzRDQlMjZkJTNET2tYb0hVZHBZRUlrQ0NxeWxOYmg5Xy5RMEliekwxUWxnLjBxdVEtLSUyNnMlM0RpaCUyNmklM0RCOG5lYU00WXU5MHNmN3AueFlkYQRjc3JjcHZpZANmR29BYVRJd05pNi5RSVhhVmtESzBBQm1Oamd1TVFBQUFBQ1hSWklyBGZyA3locy1vbXItMDAxBGZyMgNzYS1ncARncHJpZANPVmx0OE1YTFNuR0trNkZQcTViX0hBBG10ZXN0aWQDbnVsbARuX3N1Z2cDMARvcmlnaW4DaW1hZ2VzLnNlYXJjaC55YWhvby5jb20EcG9zAzAEcHFzdHIDBHBxc3RybAMEcXN0cmwDMzEEcXVlcnkDRG9jdG9yIERvb20gZHVwbGljYXRpbmcgaGltc2VsZgR0X3N0bXADMTQ4NDYwMTAxNAR2dGVzdGlkA251bGw-?gprid=OVlt8MXLSnGKk6FPq5b_HA&pvid=fGoAaTIwNi6.QIXaVkDK0ABmNjguMQAAAACXRZIr&p=Doctor+Doom+duplicating+himself&fr=yhs-omr-001&fr2=sb-top-images.search.yahoo.com&ei=UTF-8&n=60&x=wrt&type=86311436&hsimp=yhs-001&hspart=omr#id=8&iurl=http%3A%2F%2Fblog.mysanantonio.com%2Fgeekspeak%2Ffiles%2F2012%2F03%2Fdoctor_vs__doctor_by_docshaner-d36i641.jpg&action=click

He can create more of himself out of nothing

File: PermissionsEx/permissions.yml (hunk 6 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: http://blog.mysanantonio.com/geekspeak/files/2012/03/doctor_vs__doctor_by_docshaner-d36i641.jpg

Comment (adambash) [inline PermissionsEx/permissions.yml]: he can create more of himself out of nothing

File: PermissionsEx/permissions.yml (hunk 7 above)
Comment (adambash) [inline PermissionsEx/permissions.yml]: http://blog.mysanantonio.com/geekspeak/files/2012/03/doctor_vs__doctor_by_docshaner-d36i641.jpg
He can create more of himself out of nothing

**Pull request merged**